bin/v-change-sys-ip-owner

```diff
@@ -31,6 +31,12 @@ is_ip_valid "$ip"
 is_ip_key_empty '$U_WEB_DOMAINS'
 is_ip_key_empty '$U_SYS_USERS'
 
+ip_status="$(get_ip_value '$STATUS')"
+
+if [ "$user" != "admin" ] && [ "$ip_status" == "shared" ]; then
+        check_result "$E_INVALID" "Only the 'admin' user can have a shared ip"
```

Comment:
Correcting a previous invalid status with `ip_status="dedicated"` here would be better than aborting

Reply:
should be good now.